fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -680,7 +670,22 @@ public String toSql() {
         }
 
         if (defaultValue.isSet) {
-            sb.append(defaultValue.toSql()).append(" ");
+            sb.append("DEFAULT ");
+            if (defaultValue.value != null) {
+                if (typeDef.getType().getPrimitiveType() != PrimitiveType.BITMAP
+                        && typeDef.getType().getPrimitiveType() != PrimitiveType.HLL) {
+                    if (defaultValue.defaultValueExprDef != null) {
+                        sb.append(defaultValue.value);
+                    } else {
+                        sb.append("\"").append(SqlUtils.escapeQuota(defaultValue.value)).append("\"");
+                    }
+                } else if (typeDef.getType().getPrimitiveType() == PrimitiveType.BITMAP) {
```

Comment:
how to handle HLL_EMPTY default value?

Reply:
when analyze, it will return error
```
        if (type.getPrimitiveType() == PrimitiveType.HLL) {
            if (defaultValue.isSet) {
                throw new AnalysisException("Hll type column can not set default value");
            }
            defaultValue = DefaultValue.HLL_EMPTY_DEFAULT_VALUE;
        }
```